istioctl/pkg/proxyconfig/proxyconfig.go

```diff
@@ -581,6 +583,9 @@ func workloadConfigCmd(ctx cli.Context) *cobra.Command {
   # Retrieve workload summary
   kubectl exec -it $ZTUNNEL -n istio-system -- curl localhost:15000/config_dump > ztunnel-config.json
   istioctl proxy-config workloads --file ztunnel-config.json
+
+  # Retrieve workload summary for a specific namespace
+  istioctl proxy-config workloads <ztunnel-name[.namespace]> --workload-namespace foo
```

Comment:
Not sure if it's only me but I think `workload-namespace` is a bit confusing because I then think which workload is it relevant to.
I would prefer a plural `--workloads-namespace`.

Reply:
The plural seems to be more intuitive. cc @bleggett